### DISCUSSION TITLE
Is there way to let the bot follow me intead of come to my location with pathfinder.

### DISCUSSION THREAD

**Discussion by Queirbeer** [PrismarineJS/mineflayer#1931] [Q&A]
Hey i have a question. Is there a way I can let the bot follow me with pathfinder. I thought I could use a while statement but that doesnt work. Any other solutions?

**Comment by TheBlueBurger** [comment]
new GoalFollow(entity)

**Comment by Queirbeer** [comment]
@BlueBurgersTDD could you maybe give a little example, cuz Im new in this.

**Comment by flowl** [comment]
There are examples in PrismarineJS/mineflayer-statemachine

https://github.com/PrismarineJS/mineflayer-statemachine

https://github.com/PrismarineJS/mineflayer-statemachine/blob/master/src/behaviors/behaviorFollowEntity.ts
